include/cling/Interpreter/Interpreter.h

```diff
@@ -613,6 +613,7 @@ namespace cling {
 
     clang::CompilerInstance* getCI() const;
     clang::Sema& getSema() const;
+    IncrementalParser& getIncrParser() const { return *m_IncrParser; }
```

Comment:
We don't want to publish the `IncrementalParser`. Can you implement this differently, e.g. by moving the implementation out of the `IncrementalParser` or by creating a facade interface in `Interpreter`?